### PR TITLE
[F/E] showing only particular categories

### PR DESCRIPTION
**Pull request by tobob**
The goal of this PR is to introduce way to filter out results per categories

Screenshot 2022-11-21 at 14 27 08
Screenshot 2022-11-21 at 14 27 12

### CODE REVIEW

**Review by shirve — APPROVED** [review]
Looks good, just one question - what is the point of having `categoriesToFilter` typed as `array` if it only ever contains just one value?

**Comment by tobob** [comment]
> Looks good, just one question - what is the point of having `categoriesToFilter` typed as `array` if it only ever contains just one value?

good point, if we will decide to change picker to handle multiple categories, then this is easier to implement it

**Comment by Matesty** [comment]
Looks good, just one issue I have noted:

- in fullscreen mode change the category to client/coding 
- then change to a narrower resolution e.g. half of the screen
- then open all categories

Zrzut ekranu_20221122_102700

File: team-fingerprints-web/src/components/Chart/Chart.tsx [hunk 1]
```diff
@@ -60,7 +71,7 @@ const Chart: FC<IProps> = ({
   useEffect(() => {
     setWidth(ref.current?.clientWidth || 0);
     setHeight(ref.current?.clientHeight || 0);
-  }, [screenWidth]);
+  }, [screenWidth, categoriesToFilter?.[0]]); //Important! need to be here to recompute height when new filters applied
```
Comment (shirve) [inline team-fingerprints-web/src/components/Chart/Chart.tsx:74]: Is targeting first array element necessary here? 
Might cause some issues in the future if the picker will be changed to handle multiple categories and someone forgets to change this dependency
```suggestion
  }, [screenWidth, categoriesToFilter]); //Important! need to be here to recompute height when new filters applied
```

**Pull request merged**